DataProcessor/DataUnpacker.h

```diff
@@ -42,17 +42,25 @@ class DataUnpacker : public QObject
     Q_PROPERTY(bool door MEMBER door NOTIFY dataChanged);
     Q_PROPERTY(bool mcu_check MEMBER mcu_check NOTIFY dataChanged);
     Q_PROPERTY(bool imd_status MEMBER imd_status NOTIFY dataChanged);
-    Q_PROPERTY(bool mps_enable MEMBER mps_enable NOTIFY dataChanged);
+    Q_PROPERTY(bool discharge_enable MEMBER discharge_enable NOTIFY dataChanged);
     Q_PROPERTY(bool mppt_contactor MEMBER mppt_contactor NOTIFY dataChanged);
     Q_PROPERTY(bool motor_controller_contactor MEMBER motor_controller_contactor NOTIFY dataChanged);
     Q_PROPERTY(bool low_contactor MEMBER low_contactor NOTIFY dataChanged);
-    Q_PROPERTY(bool bms_canbus_failure MEMBER bms_canbus_failure NOTIFY dataChanged);
+    Q_PROPERTY(bool bms_can_heartbeat MEMBER bms_can_heartbeat NOTIFY dataChanged);
     Q_PROPERTY(bool voltage_failsafe MEMBER voltage_failsafe NOTIFY dataChanged);
     Q_PROPERTY(bool current_failsafe MEMBER current_failsafe NOTIFY dataChanged);
-    Q_PROPERTY(bool supply_power_failsafe MEMBER supply_power_failsafe NOTIFY dataChanged);
-    Q_PROPERTY(bool memory_failsafe MEMBER memory_failsafe NOTIFY dataChanged);
+    Q_PROPERTY(bool input_power_supply_failsafe MEMBER input_power_supply_failsafe NOTIFY dataChanged);
+    Q_PROPERTY(int mc_status MEMBER mc_status NOTIFY dataChanged);
     Q_PROPERTY(bool relay_failsafe MEMBER relay_failsafe NOTIFY dataChanged);
     Q_PROPERTY(bool bps_fault MEMBER bps_fault NOTIFY dataChanged);
+    Q_PROPERTY(bool cell_balancing_active MEMBER cell_balancing_active NOTIFY dataChanged);
+    Q_PROPERTY(bool mcu_hv_en MEMBER mcu_hv_en NOTIFY dataChanged);
+    Q_PROPERTY(bool charge_interlock_failsafe MEMBER charge_interlock_failsafe NOTIFY dataChanged);
+    Q_PROPERTY(bool thermistor_b_value_table_invalid MEMBER thermistor_b_value_table_invalid NOTIFY dataChanged);
+    Q_PROPERTY(bool mcu_stat_fdbk MEMBER mcu_stat_fdbk NOTIFY dataChanged);
+    Q_PROPERTY(bool dcdc_valid MEMBER dcdc_valid NOTIFY dataChanged);
+    Q_PROPERTY(bool supplemental_valid MEMBER supplemental_valid NOTIFY dataChanged);
+    Q_PROPERTY(bool charge_enable MEMBER charge_enable NOTIFY dataChanged);
```

Comment:
```suggestion
    Q_PROPERTY(bool input_power_supply_failsafe MEMBER input_power_supply_failsafe NOTIFY dataChanged);
    Q_PROPERTY(bool relay_failsafe MEMBER relay_failsafe NOTIFY dataChanged);
    Q_PROPERTY(bool cell_balancing_active MEMBER cell_balancing_active NOTIFY dataChanged);
    Q_PROPERTY(bool charge_interlock_failsafe MEMBER charge_interlock_failsafe NOTIFY dataChanged);
    Q_PROPERTY(bool thermistor_b_value_table_invalid MEMBER thermistor_b_value_table_invalid NOTIFY dataChanged);
    Q_PROPERTY(bool charge_enable MEMBER charge_enable NOTIFY dataChanged);
    Q_PROPERTY(bool bps_fault MEMBER bps_fault NOTIFY dataChanged);
    Q_PROPERTY(bool dcdc_valid MEMBER dcdc_valid NOTIFY dataChanged);
    Q_PROPERTY(bool supplemental_valid MEMBER supplemental_valid NOTIFY dataChanged);
    Q_PROPERTY(bool mcu_hv_en MEMBER mcu_hv_en NOTIFY dataChanged);
    Q_PROPERTY(bool mcu_stat_fdbk MEMBER mcu_stat_fdbk NOTIFY dataChanged);
    Q_PROPERTY(int mc_status MEMBER mc_status NOTIFY dataChanged);
```
A bit nit-picky, but can you please reorder the Q_PROPERTYs like this so that similar values are grouped together?